docs/start/framework/react/server-functions.md

```diff
@@ -472,6 +511,65 @@ export const getServerTime = createServerFn({ method: 'GET' }).handler(
 )
 ```
 
+For more advanced control over responses, especially with streaming data, you can use the `response: 'raw'` option:
```

Comment:
please change the above example as well. you MUST specify `response: 'raw'` when you want to return a raw response

Reply:
Done

> please change the above example as well. you MUST specify `response: 'raw'` when you want to return a raw response